docs/data/sql_functions_zh.yml

```diff
@@ -0,0 +1,820 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License
+
+comparison:
+  - sql: value1 = value2
+    table: value1 === value2
+    description: 如果 value1 等于 value2 返回 `TRUE`；如果 value1 或者 value2 为 NULL 返回 `UNKNOW`。
```

Comment:
Constants in expression like \``TRUE`\`  are typed with \` in this translation but without \` in  the file `sql_functions.yml`.
Should we keep consistent format or make a new PR thread to update it?
@wuchong  @ZhijieYang  What's your opinion?

Reply:
It's my own opinion to use `` because I consider without some difference it is hard to distinguish between the doc and code. If it is necessary to be the same as `sql_functions.yml`, I will make a new PR to commit after all the modify.

Reply:
Maybe we should keep values in code like `NULL` or results of method calling be wrappered with `

Reply:
great idea